source/op/_tabulate_grad.py

```diff
@@ -8,12 +8,22 @@
 from deepmd.env import tf
 # from deepmd.DescrptSeATabulate import last_layer_size
 
-@ops.RegisterGradient("TabulateFusion")
-def _tabulate_fusion_grad_cc (op, dy):    
-    dy_dx, dy_df = op_module.tabulate_fusion_grad(op.inputs[0], op.inputs[1], op.inputs[2], op.inputs[3], dy, op.outputs[0])
+@ops.RegisterGradient("TabulateFusionSeA")
```

Comment:
This rename will cause model compatibility issue, though.

Reply:
The op `TabulateFusion` are still registered in the `tabulate_multi_device.cc` just like the `DescrptSeA` in `prod_env_mat_multi_device.cc`. So I think this change should not be able to affect the inference process. For the `init-frz-model` process, I will add the Decorators `@ops.RegisterGradient("TabulateFusion")`.

Reply:
Backward compatibility is fine but forward compatibility is broken. Do we need to bump model_version?

Reply:
I think you are right. How should I bump the model_version?

Reply:
update the file source/config/MODEL_VER

Reply:
got it